blog.html

```diff
@@ -22,6 +22,286 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_400_release_announcement"></a>
+                    <a href="#apache_kafka_400_release_announcement">Apache Kafka 4.0.0 Release Announcement</a>
+                </h2>
+                17 March 2025 - David Jacot
+                
+                <p>
+                    We are proud to announce the release of Apache Kafka&#174; 4.0 This release contains many new
+                    features and improvements. This blog post will highlight some of the more prominent features.
+                    For a full list of changes, be sure to check the <a href="https://archive.apache.org/dist/kafka/4.0.0/RELEASE_NOTES.html">release notes</a>. 
+                </p>
+                <p>
+                    See the <a href="https://kafka.apache.org/documentation.html#upgrade_4_0_0">Upgrading to 4.0 from any versions 0.8.x through 3.9.x</a> section in the
```

Comment:
the title of "upgrade_4_0_0" is `Upgrading to 4.0.0 from any version 3.3.x through 3.9.x`

Reply:
I would move this and "for a full list of changes" to the end.